Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -593,18 +594,76 @@ open class Kommunicate: NSObject,Localizable, KMPreChatFormViewControllerDelegat
         }
     }
     
-    public func userSubmittedResponse(name: String, email: String, phoneNumber: String, password: String) {
+    /**
+    launches the PreChat Lead Collection with custom payload
+
+     - Parameters:
+     - appID: User's application ID.
+     - inputList: list of LeadCollectionField objects to create a form in pre Chat.
+     - viewController: ViewController from which the pre-chat form view will be launched.
+     - prechatCompletion:  Callback to inform prechat launched successfully or not
+     - completion: Callback to pass form response or error
```

Comment:
Change the name to `onButtonTap` or something to make it more clear.

Reply:
named as `onFormCompletion`